srcCpp/Infrastructure_micro.cxx

```diff
@@ -88,6 +89,10 @@ unsigned long long PerftestClock::getTimeUsec()
     QueryPerformanceCounter(&ticks);
     return ticks.QuadPart / (unsigned long long) (_frequency /1000000.0);
 
+    #endif
```

Comment:
As a suggestion, for more readable code
`#endif  /* RTI_WIN32 */`

Reply:
Done